README.md

```diff
@@ -16,12 +16,50 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
-npm install @stoplight/spectral
+### Local Installation
```

Comment:
I think, overall, the top of our readme is getting too verbose with all the installation stuff. It's pushing down actual simple usage. I'll open a separate PR to cleanup readme a bit separately though.

Reply:
Yeah we can swap usage and install, that’s pretty common. Figure out if it does what you want before you figure out how to get it. I think I made it this current order, but then it grew.